keyboards/splitography/keymaps/beaklmu/common/keycode_functions.h

```diff
@@ -0,0 +1,660 @@
+
+// .......................................................... Keycode Primitives
+
+// register simple key press
+void tap_key(uint16_t keycode)
+{
+  register_code  (keycode);
+  unregister_code(keycode);
+}
```

Comment:
This is not needed.  `tap_code` is a core feature/function that does the same thing.